ADOPTERS.md

```diff
@@ -20,6 +20,7 @@
 | Integration | osbuild-operator | 2022 | [link](https://github.com/project-flotta/osbuild-operator) | OSBuild-Operator uses KubeVirt to provision its internal worker VMs. |
 | Integration | PITS Global Data Recovery Services | 2023 | [link](https://www.pitsdatarecovery.net/) | KubeVirt allows us to manage highly-loaded VMs and containers from one place. |
 | Integration | Kasten by Veeam | 2022 | [link](https://docs.kasten.io/latest/usage/openshift_virtualization.html?highlight=kubevirt) | [Kasten K10](https://www.kasten.io/product/) manages KubeVirt and Red Hat OpenShift Virtualization VMs seamlessly for policy based automation of backup, disaster recovery, and application mobility across different Kubernetes versions, distributions, storage providers, and clouds. |
+| End-User | The Linux Foundation - Training and Certification | 2022 | [link](https://training.linuxfoundation.org/) | The Linux Foundation uses KubeVirt to manage virtual machines in Kubernetes that are used as lab or certification environments. |
```

Comment:
@relaxdiego This is great to see!
I agree with @brianmcarey's comment - can you please move this up to line 15 so that it is grouped with the other End users?

Reply:
Move further up next to the other end users. Thanks!